### ISSUE TITLE
Identify spam/fake bugs

### ISSUE THREAD

**Issue by emceeaich** [mozilla/bugbug#1013]
We get too many spam bugs filed by students at schools with instructors telling them to learn how to use Bugzilla by filing junk bugs here. 

Most of the junk bugs have been moved to the Invalid Bugs::General component. 

Can we add a model we can run against incoming bugs that can mark as "potential spam" bug and divert to a moderator for review?

**Comment by marco-c** [comment]
Yes, I think it should be feasible! We have plenty of "good" bugs (we can just use past RESOLVED FIXED bugs as examples of "good" bugs), and we can get "bad" bugs by using "Invalid Bugs::General" (hoping there are enough to train the model).

**Comment by ayush-1506** [comment]
This shouldn't be very tough to implement. @marco-c Please let me know if I should go ahead with this. (I noticed the good-first-bug label, so maybe I should let someone else work on this?)

**Comment by m-agboola** [comment]
@marco-c I'm interested in working on this if no one is assigned to it

**Comment by marco-c** [comment]
@ayush-1506 you can work on it, don't worry (I mark them as good-first-issues because they can be worked on in isolation, but it doesn't mean only newcomers can work on them).

@m-agboola you can work on it too, any issue is unassigned until there is a PR open to fix it. It is not the simplest issue though, you might want to start with another one.

**Comment by m-agboola** [comment]
Thanks @marco-c I understand and would give it a try!

**Comment by ayush-1506** [comment]
@marco-c Correct me if I'm wrong:
Good bugs are the ones with `STATUS` as `FIXED` or `RESOLVED`.
Bad bugs are the ones where `Invalid Bugs` was added at any time in the history, right?

If so, there aren't a lot of bad bugs. Maybe we should consider other indicative fields too?

**Comment by marco-c** [comment]
I think the problem is simply that we are not downloading the bad bugs in our DB, as "Invalid Bugs" is not in the list of considered components at https://github.com/mozilla/bugbug/blob/a36edbbe14bd1590d42ac5b171a17c894c066301/bugbug/bugzilla.py#L25.

**Comment by ayush-1506** [comment]
@marco-c So we'd need to add it to the products, regenerate the bugs db and upload it to tc, right? Should this be a two step PR? (First one adding the label to the products and the second one implementing the model)

**Comment by marco-c** [comment]
It can be a two step PR or a single PR with two separate commits.
Note: you can add new bugs to the DB locally, by simply running the bug_retriever script, so you can start working on the model directly without waiting on the next release for TC to do it.

**Comment by marco-c** [comment]
@ayush-1506 still working on this?

**Comment by emceeaich** [comment]
Just to clarify, a bad bug would be anything in the Invalid Bugs product, or any bug that is resolved as INVALID.

**Comment by ayush-1506** [comment]
@marco-c I think this issue can be closed, right? Both the points mentioned in the above comment have been added. Should I open a new issue to address the other parts (adding number of words as a feature)?

**Comment by marco-c** [comment]
@ayush-1506 I'd prefer to keep this open until we have a working model, the current one doesn't have any feature which is specific for it and so it's results are not good IIRC.

**Comment by marco-c** [comment]
We have a feature specific for this model now (thanks @ayush-1506!), and I created a project to track further improvements and work needed for this: https://github.com/mozilla/bugbug/projects/9.
